loader_hub/slack/base.py

```diff
@@ -116,10 +116,18 @@ def _read_channel(self, channel_id: str, reverse_chronological: bool) -> str:
                 # conversations.history returns the first 100 messages by default
                 # These results are paginated,
                 # see: https://api.slack.com/methods/conversations.history$pagination
-                result = self.client.conversations_history(
-                    channel=channel_id,
-                    cursor=next_cursor,
-                )
+                if self.earliest_date_timestamp is None:
+                    result = self.client.conversations_history(
+                        channel=channel_id,
+                        cursor=next_cursor,
```

Comment:
shouldn't this also include latest?

Reply:
we are filtering it based on earliest date and so the code is modified accordingly.